packages/shared/src/lib/localLanguage.ts

```diff
@@ -1,50 +1,41 @@
 export const LocalLanguageBannerTemplateName = 'EuropeMomentLocalLanguageBanner';
-const LocalLanguageBannerTestName = 'LOCAL-LANGUAGE';
-const LocalLanguageBannerVariant = 'CONTROL';
-const LocalLanguageEpicTestName = 'LOCAL-LANGUAGE';
-const LocalLanguageEpicVariant = 'CONTROL';
 
 export type LocalLanguage = {
+    testName: string;
+    variantName: string;
     bannerHeader?: string;
-    epicHeader?: string;
-    epicParagraphs?: string[];
-    epicHighlightedText?: string;
 };
```

Comment:
From what I can tell this type may only need the bannerHeader so could also be renamed. Have also removed the export as it looks to be only used in this file
```
type BannerHeaderCopy = {
	bannerHeader: string | undefined;
};
```

Reply:
Ok, removed

Reply:
The export statement here can be removed and the type's name (which is the return type of the getter function) is a bit vague

Reply:
Yes, should of spotted that, renamed to LocalLanguageBannerHeader too.